docs/public/svg/amplify-ui-logo.svg

```diff
@@ -0,0 +1,10 @@
+<svg class="docs-logo-svg" viewBox="0 0 644 125" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Did you mean to use this in the README? It does not appear to be used.

Reply:
I used it in a previous commit, but since it doesn't work with Github dark mode, I didn't use it in the new commit. Let me remove it.

Reply:
Removed.